scaaml/capture/aes/crypto_alg.py

```diff
@@ -119,7 +121,7 @@ def __next__(self):
                 # AcqKeyTextPatternScaaml.new_pair raises StopIteration itself.
                 kt_pair = self._ktp.new_pair()
                 # Allow the same iteration as using resume_kti.
-                return list(kt_pair[0]), list(kt_pair[1])
+                return {"keys": list(kt_pair[0]), "texts": list(kt_pair[1])}
```

Comment:
maybe that's just personal preference, but a namedtuple here would look nicer than a dict for the return value of the iterator

Reply:
We get nicer unpacking. Agreed.